initramfs/debian/rules

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ 
+
+override_dh_auto_test:
+	@set -e; for f in scripts/*; do \
+	    echo "Checking shell syntax of $$f"; \
+	    sh -n $$f; \
```

Comment:
Having shellcheck here would be nice (probably needs some excludes because the file is sourced).